packages/commonwealth/client/scripts/views/components/ShareSection/ShareOptionComponent.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { ShareOption } from './ShareOption';
+
+export const ShareOptionComponent = (props: ShareOption) => {
```

Comment:
ShareOption and ShareOptionComponent does not make sense. 
Lets have just ShareOption which will be component and will have interface defined as well. Remove ShareOptionComponent.tsx

Reply:
Let me know if you're good with this.

I renamed it to ShareOptionButton

I think there was a misunderstanding.

useShareOptions and ShareOption was there before.  

I think useShareOptions() and ShareOption is elegant because we keep it data driven.

But ShareOptionComponent probably wasn't the best name. 

ShareOptionButton I think is better.

I think this is a bit cleaner.  

Let me know if you're good with this.

Reply:
thanks for explanation, thats fine